lang/LangSource/PyrObject.cpp

```diff
@@ -1169,7 +1169,7 @@ void buildBigMethodMatrix()
 	double t0 = elapsedTime();
 #endif
 
-	const int hw_concurrency    = thread::hardware_concurrency();
+	const int hw_concurrency    = std::thread::hardware_concurrency();
```

Comment:
Thanks. I did that intentionally. I do not know C++, so I thought declared type will not work as a namespace. (Or you will not be able to call its class methods, or whatever they called). Will fix it tomorrow.